src/components/MenuBar.vue

```diff
@@ -77,8 +82,21 @@ export default {
 	data() {
 		return {
 			searchQuery: "",
+			avatarUrl: "", // Inicializa o avatarUrl
 		};
 	},
+	async created() {
+		const { data: user } = await supabase.auth.getUser();
+
+		// Tenta carregar a URL do avatar do localStorage
+		const savedUser = JSON.parse(localStorage.getItem("user"));
+		if (savedUser && savedUser.avatarUrl) {
```

Comment:
Não estou lembrado se isso é suportado pelo vue.js na versão que estamos utilizando nesse projeto, mas existe uma short-syntax para ter apenas uma condicional ao invés de duas utilizando um optional chaining `?.`:

```suggestion
		if (savedUser?.avatarUrl) {
```

Reply:
![image](https://github.com/user-attachments/assets/fc4026bd-4042-4acf-9ff6-b35604106afb)
obrigado pela dica monstro